src/sentry/api/endpoints/group_suspect_releases.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+
+from django.db.models import Q
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases import GroupEndpoint
+from sentry.api.helpers.environments import get_environments
+from sentry.api.serializers import GroupSerializerSnuba, serialize
+from sentry.models.deploy import Deploy
+from sentry.models.grouphistory import GroupHistory, GroupHistoryStatus
+from sentry.models.release import Release
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+
+class GroupSuspectReleasesEndpoint(GroupEndpoint, EnvironmentMixin):
+    enforce_rate_limit = True
+    rate_limits = {
+        "GET": {
+            RateLimitCategory.IP: RateLimit(5, 1),
+            RateLimitCategory.USER: RateLimit(5, 1),
+            RateLimitCategory.ORGANIZATION: RateLimit(5, 1),
+        },
+    }
+
+    def get(self, request: Request, group) -> Response:
+        """
+        Retrieve Suspect Releases for an Issue
+        ``````````````````````````````````````
+
+        Return the suspect releases for an issue. Suspect releases are defined
+        as releases which caused the first seen issue or the latest regression
+        during the active release window (1 hour).
+        :pparam string issue_id: the ID of the issue to retrieve.
+        :auth: required
+        """
+
+        organization = group.project.organization
+        environment_ids = [e.id for e in get_environments(request, organization)]
+
+        data = serialize(group, request.user, GroupSerializerSnuba(environment_ids=environment_ids))
```

Comment:
Do we need to actually include the group here? I think just returning the list of suspect releases should be enough for our purposes. Since this will be called on the issue details page, we will have already made a call to fetch group details before.

Reply:
Did this get addressed?